app/views/layouts/application.html.erb

```diff
@@ -23,9 +23,19 @@
 
   <!-- Stylesheets -->
   <%= stylesheet_link_tag 'application', media: 'all' %>
+  <% if !@extra_css.blank? %>
+    <% @extra_css.each do |css| %>
+  <%= css %>
+    <% end %>
+  <% end %>
 
   <!-- Scripts -->
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => false %>
+  <% if !@extra_js.blank? %>
+    <% @extra_js.each do |js| %>
+  <%= js %>
```

Comment:
This is intentional so it lines up with the indentation of the markup inside the `<head>` element. If there is a way to do it so it looks good in both places, that'd be awesome, but I like the public HTML to look nicer than indentation here if I have to choose I guess.
Though the rest of the HTML indentation is a total mess :-(